components/VendorCard.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+import ButtonLink from '../components/ButtonLink';
+import { Vendor } from "typings";
+
+interface VendorCardProps {
+    vendor: Vendor;
+}
+
+function VendorCard(props: VendorCardProps): JSX.Element {
+    const { vendor } = props;
+    const {
+        name,
+        logoUrl,
+        url
+    } = vendor;
+
+    return (
+        <div className="vendor-card">
+            <div className="image">
+                <img src={logoUrl === undefined || logoUrl === '' ? '/images/empty-base-kit-illu.svg' : logoUrl} alt="Vendor Logo" />
```

Comment:
base kit illu in vendor? Misschien is er een betere oplossing. Naam ipv logo bijv?

Reply:
Jep, zou naam kunnen doen, maar dan komt er gewoon twee keer de Vendor naam te staan. Dat is misschien een beetje lame?